modules/ipi-install-out-of-band-management.adoc

```diff
@@ -6,8 +6,18 @@
 [id="out-of-band-management_{context}"]
 = Out-of-band management
 
-Nodes will typically have an additional NIC used by the baseboard management controllers (BMCs). These BMCs must be accessible from the provisioner node.
+Nodes typically have an additional NIC used by the baseboard management controllers (BMCs). These BMCs must be accessible from the provisioner node.
 
 Each node must be accessible via out-of-band management. When using an out-of-band management network, the provisioner node requires access to the out-of-band management network for a successful {product-title} installation.
 
-The out-of-band management setup is out of scope for this document. We recommend setting up a separate management network for out-of-band management. However, using the provisioning network or the baremetal network are valid options.
+The out-of-band management setup is out of scope for this document. You can use a separate management network for out-of-band management. However, using the provisioning network or the baremetal network are also valid options.
```

Comment:
Is this a bad time to mention that using the baremetal network for this is a terrible idea for security reasons, even though it will work?
I preferred when we at least recommended doing it correctly.

Not sure if this is the place, but ideally the management network should be firewalled off so that only the provisioning host and control plane can access it. That's hinted at in https://docs.openshift.com/container-platform/4.12/installing/installing_bare_metal_ipi/ipi-install-prerequisites.html#network-requirements-out-of-band_ipi-install-prerequisites but not really stated explicitly.

Reply:
Thanks Zane. We try to avoid the word "recommend" in the docs. So I've rephrased the sentence again. However, if you think in this instance we are confident in the recommendation I can change it back to the original.

FYI - the IBM style guide around the word _recommend_ : "Avoid making recommendations or claims about performance, benefits, or future support that could create a potential marketing or legal problem for IBM."